front/src/modules/ui/board/components/BoardColumn.tsx

```diff
@@ -60,6 +60,27 @@ const StyledAmount = styled.div`
   margin-left: ${({ theme }) => theme.spacing(2)};
 `;
 
+const StyledNumChildren = styled.div`
+  align-items: center;
```

Comment:
could you rewrite this CSS using the minimum of properties + using theme for all propreties (border-radius, font-size, gap, line-height, etc.) I don't think we need that much properties!

Reply:
I have removed the excess properties and have tried to use theme to describe all the properties. Let me know if I should change anything.
Thanks!!